wire-ios-data-model/Tests/TestPlans/AllTests.xctestplan

```diff
@@ -9,12 +9,92 @@
     }
   ],
   "defaultOptions" : {
+    "commandLineArgumentEntries" : [
+      {
+        "argument" : "-com.apple.CoreData.SQLDebug 1",
+        "enabled" : false
+      },
+      {
+        "argument" : "-com.apple.CoreData.ConcurrencyDebug 1"
+      }
+    ],
+    "environmentVariableEntries" : [
+      {
+        "key" : "SQLITE_ENABLE_THREAD_ASSERTIONS",
+        "value" : "1"
+      },
+      {
+        "enabled" : false,
+        "key" : "SQLITE_AUTO_TRACE",
+        "value" : "1"
+      },
+      {
+        "key" : "SQLITE_ENABLE_FILE_ASSERTIONS",
+        "value" : "1"
+      }
+    ],
     "testTimeoutsEnabled" : true
   },
   "testTargets" : [
     {
       "skippedTests" : [
-        "MLSServiceTests"
+        "MLSServiceTests\/test_AddingMembersToConversation_ClaimKeyPackagesFails()",
```

Comment:
Enables the whole file and disables all the existing tests instead, so that the newly added tests can be executed.

Reply:
Do you know why the tests were disabled?

Reply:
Because they are flaky.

Reply:
Will be addressed in https://wearezeta.atlassian.net/browse/WPB-5481

Reply:
any ETA @David-Henner ?

Reply:
some time this week